pages/assessment.tsx

```diff
@@ -39,7 +44,7 @@ const Assessment: NextPage = () => {
   const [taskNumber, setTaskNumber] = useState<any>("");
   const [taskTitle, setTaskTitle] = useState<string>("");
   const [currentPage, setCurrentPage] = useState<number>(1);
-  const [maxItemsPerPage, setMaxItemsPerPage] = useState<number>(4); //max items set to 4 as default
+  const [max, setMaxItemsPerPage] = useState<string>("4"); //max items set to 4 as default
```

Comment:
Just a small thing: maybe `max, setMax` or `maxItemsPerPageString, setMaxItemsPerPageString` ?
so the two corresponds